etl/src/main/java/org/pih/hivmigration/etl/sql/VitalsMigrator.groovy

```diff
@@ -0,0 +1,85 @@
+package org.pih.hivmigration.etl.sql
+
+class VitalsMigrator extends SqlMigrator {
+    @Override
+    def void migrate() {
+
+        executeMysql("Create vitals staging table", '''
+            CREATE TABLE hivmigration_vitals (
+              source_encounter_id int,
+              sign varchar(30),
+              result double,
+              result_unit varchar(100),
+              source_patient_id int,
+              KEY `source_encounter_idx` (`source_encounter_id`)
+            );
+        ''')
+
+        // TODO: should we do something with the 563 vitals with encounter type 'food_study'
+        //      or the 3 with type patient_contact?
+        loadFromOracleToMySql('''
+            INSERT INTO hivmigration_vitals (source_encounter_id, sign, result, result_unit, source_patient_id)
+            VALUES (?, ?, ?, ?, ?)
+        ''', '''
+            select 
+                v.ENCOUNTER_ID, 
+                v.SIGN, 
+                v.RESULT, 
+                v.RESULT_UNIT,
+                e.PATIENT_ID
+            from HIV_EXAM_VITAL_SIGNS v, HIV_ENCOUNTERS e
+            where v.ENCOUNTER_ID = e.ENCOUNTER_ID 
+            and SIGN in ('height', 'weight', 'bmi')
```

Comment:
Can we just bang out this entire table migration now, instead of just migrating these 3 values?  There are not that many options, and I would expect we already have a clear migration for all of them.

select sign from HIV_EXAM_VITAL_SIGNS group by sign order by sign;

height,weight,bmi
+
blood_pressure_dias, blood_pressure_sys, heart_rate, respiration_rate, temperature

So we just need to add in migration for 5 additional concepts, which are all in our Vitals app in OpenMRS already.

Can we do this rather than leave this as a pending todo?  Especially since we are migrating to a Vitals encounter.